content/CORE/UIReference/JailsPluginsVMs/Plugins/PluginsScreens.md

```diff
@@ -123,3 +127,5 @@ For information on Plugins and Jails screens and how to add or manage jails see:
 [Managing Jails]({{< relref "/CORE/CORETutorials/JailsPluginsVMs/Jails/ManagingJails.md" >}})
 
 [Setting Up Jail Storage]({{< relref "/CORE/CORETutorials/JailsPluginsVMs/Jails/SettingUpJailStorage.md" >}})
+
+{{< taglist tag="coreapps" limit="10" title="Related Plugins Articles" >}}
```

Comment:
The taglist shortcode is a bit confusing. you used tag="coreapps" but have decided to use title="Related Plugins Articles", why not just use tag="coreplugins" instead? Also if using only one taglist shortcode you don't need to include the title="" parameter.

Reply:
The tag is now "coreplugins." The taglist on PluginsScreens.md no longer has the title parameter. The title parameter appears on GettingStarted\Applications.md which this article links to and which has more than one taglist shortcode.